extension/controller/backgroundScript.js

```diff
@@ -18,14 +19,19 @@ const messageListener = async function(message, sender) {
     let webNavigationCompletedLoad = null;
     switch (message.command) {
         case "detectPageLanguage":
+            if (!modelFastText) break;
```

Comment:
how long it might take to load the fasttext module? should we figure out some waiting mechanism to start mediator only after it's loaded or it's good enough?

Reply:
It takes less than a second on my computer, so it will be loaded when the addon is either installed or when the browser is started, and the flag above prevents the mediator of starting if it hasn't loaded yet. That's enough